src/components/NavBars/TitleBar/TitleBar.tsx

```diff
@@ -170,14 +170,14 @@ const TitleBar: React.FC<TitleBarProps> = ({
 						<IconButton
 							iconUri={arrowBackIcon}
 							onClick={onBack}
-							style={{ height: 32, width: 32 }}
+							style={{ height: 42, width: 42 }}
 							disabled={!canGoBack}
 							alt={'back'}
 						/>
 						<IconButton
 							iconUri={arrowForwardIcon}
 							onClick={onForward}
-							style={{ height: 32, width: 32, marginLeft: 4 }}
+							style={{ height: 42, width: 42, marginLeft: 4 }}
```

Comment:
These buttons need to stay 32x32 per Figma.

I think you've rendered out the button SVGs with the whole FIgma frame - you just need to render the arrow itself. The arrow should be 16x16 and should slot nicely into the 32x32 container.

Reply:
I've done the update. SVG size was updated to 32x32 px and added padding to IconButton container.